tools/update-readme.js

```diff
@@ -84,20 +95,62 @@ function formatTeamMembers(members) {
  * @returns {string} The HTML for the readme.
  */
 function formatSponsors(sponsors) {
-    const nonEmptySponsors = Object.keys(sponsors).filter(tier => sponsors[tier].length > 0);
+    const nonEmptySponsors = Object.keys(sponsors).filter(
+        tier => sponsors[tier].length > 0
+    );
```

Comment:
```suggestion
    const nonEmptySponsors = Object.keys(sponsors).filter(
        tier => sponsors[tier].length);
```
no need to explicitly check if it is greater than 0.